<launch>

  <node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf">
    <param name="output_frame" value="odom_combined"/>
    <param name="freq" value="30.0"/>
    <param name="sensor_timeout" value="1.0"/>
    <param name="odom_used" value="true"/>
    <param name="vo_used" value="true"/>
    <param name="debug" value="false"/>
    <param name="self_diagnose" value="false"/>
  </node>

  <node pkg="localization" type="encoders" name="encoders">
  </node>

  <node pkg="localization" type="dummy" name="dummy">
    <param name="topic" value="/vo"/>
  </node>

 </launch>
